common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -480,13 +496,8 @@ def toggles_showing_of_discussion_topics(self):
         """
         Shows the discussion topics.
         """
-        EmptyPromise(
-            lambda: self.q(css=self._bounded_selector('.toggle-cohort-management-discussions')).results != 0,
-            "Waiting for discussion section to show"
-        ).fulfill()
-
-        # If the discussion topic section has not yet been toggled on, click on the toggle link.
-        self.q(css=self._bounded_selector(".toggle-cohort-management-discussions")).click()
+        self.q(css=self._bounded_selector(".toggle-cohort-management-discussions")).first.click()
```

Comment:
Might there need to be a change to the [click method in bok choy](https://github.com/edx/bok-choy/blob/master/bok_choy/query.py#L433-L447) instead?